testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/crossdc/ActionTokenCrossDCTest.java

```diff
@@ -89,7 +92,18 @@ public void sendResetPasswordEmailSuccessWorksInCrossDc(
       @JmxInfinispanCacheStatistics(dc=DC.SECOND, dcNodeIndex=0, cacheName=InfinispanConnectionProvider.ACTION_TOKEN_CACHE) InfinispanStatistics cacheDc1Node0Statistics,
       @JmxInfinispanChannelStatistics() InfinispanStatistics channelStatisticsCrossDc) throws Exception {
         log.debug("--DC: START sendResetPasswordEmailSuccessWorksInCrossDc");
-        
+
+        // KEYCLOAK-17584: Temporarily disable the test for 'community' profile till KEYCLOAK-17628 isn't fixed. In other words till:
+        // * The test is either rewritten to start using the new Wildfly subsystem for base metrics introduced in Wildfly 22,
+        // * Or Keycloak is able to load the Eclipse MicroProfile Metrics subsystem from the microprofile Galleon feature-pack
+        String profileName = null;
+        try (Keycloak adminClient = AdminClientUtil.createAdminClient()) {
+            profileName = adminClient.serverInfo().getInfo().getProfileInfo().getName();
+        } catch (Exception e) {
+            throw new RuntimeException("Failed to obtain profile name.", e);
+        }
+        Assume.assumeTrue("Ignoring test as product profile is not enabled", profileName.equals("product"));
```

Comment:
@martin-kanis Thanks for the review, Martin!

Regarding the failing GHA tests - I don't think they are related, since in previous two GHA runs they were passing:
* https://github.com/keycloak/keycloak/actions/runs/721914213 (13 days ago)
* https://github.com/keycloak/keycloak/actions/runs/708628022 (18 days ago)

and for the change, where they failed didn't do anything different than just rebasing to current Keycloak `master` branch.

Regarding the `ProfileAssume.assumeFeatureEnabled` call - unsure how to utilize exactly than one (since we don't need to query some feature, but specific profile). But you are correct, there's easier way how to query profile name without re-implementing `ProfileAssume` functionality. Added that simplification (to both `ActionTokenCrossDCTest.java` and to `MetricsRestServiceTest.java`) within the last push.

PTAL & let me know if that's OK

Thanks!
Jan